doc/release/1.13.0-notes.rst

```diff
@@ -29,9 +32,8 @@ Future Changes
 Compatibility notes
 ===================
 
-
 Error type changes
-~~~~~~~~~~~~~~~~~~
```

Comment:
It's not dashes the outlier, it's dashes, a new hope.

I introduced tildes back when to solve one rendering problem, and now that we are releasing on github, dashes for subtitles and stars for lists gives us better markdown compatibilit.